Microsoft.Toolkit.Uwp.SampleApp/SamplePages/Menu/Menu.bind

```diff
@@ -0,0 +1,167 @@
+<Page x:Class="Microsoft.Toolkit.Uwp.SampleApp.SamplePages.MenuPage"
+      xmlns="http://schemas.microsoft.com/winfx/2006/xaml/presentation"
+      xmlns:x="http://schemas.microsoft.com/winfx/2006/xaml"
+      xmlns:commands="using:Microsoft.Toolkit.Uwp.SampleApp.Menu.Commands"
+      xmlns:controls="using:Microsoft.Toolkit.Uwp.UI.Controls"
+      xmlns:d="http://schemas.microsoft.com/expression/blend/2008"
+      xmlns:mc="http://schemas.openxmlformats.org/markup-compatibility/2006"
+      mc:Ignorable="d">
+
+    <Page.Resources>
+        <ResourceDictionary>
+            <ResourceDictionary.MergedDictionaries>
+                <ResourceDictionary Source="MenuStyles.xaml" />
+            </ResourceDictionary.MergedDictionaries>
+
+            <commands:NewProjectCommand x:Key="NewProject" />
+            <commands:NewFileCommand x:Key="NewFile" />
+            <commands:GenericCommand x:Key="GenericCommand" />
+        </ResourceDictionary>
+
+
+    </Page.Resources>
+
+    <Grid Background="{StaticResource Brush-Grey-05}">
+        <Grid.RowDefinitions>
+            <RowDefinition Height="auto" />
+            <RowDefinition Height="auto" />
+        </Grid.RowDefinitions>
+
+        <controls:Menu HeaderButtonStyle="{StaticResource MenuHeaderButtonStyle}"
+                       MenuFlyoutStyle="{StaticResource MenuStyle}"
```

Comment:
This what happened when I removed the MenyFlyoutStyle and MenuFlyoutItem and Sub styles
![image](https://cloud.githubusercontent.com/assets/6191377/26269163/84ae9f44-3d36-11e7-90b3-92ffd9bb27c6.png)

and this what happens when I removed the MenyFlyoutItem and SubItem styles
![issuewithstyle](https://cloud.githubusercontent.com/assets/6191377/26269326/12692228-3d37-11e7-9fbf-c60b5a979a42.png)

As you can see the problem is not with the first menuItem generation but with the second